.gitignore

```
@@ -64,3 +64,4 @@ lib
 !/packages/terra-cli/tests/jest/fixtures/**/lib/
 !/packages/duplicate-package-checker-webpack-plugin/tests/jest/**/node_modules/
 .idea
+packages/terra-functional-testing/src/.DS_Store
```

Why not make update it to this so it's for the whole repo?
```suggestion
.DS_Store
```